packages/next-app/src/app/internalmanager/page.tsx

```diff
@@ -1,15 +1,26 @@
 "use client";
 import { NetworkChainId } from "@balancer-pool-metadata/shared";
+import { parseFixed } from "@ethersproject/bignumber";
 import { useEffect, useState } from "react";
-import { useAccount, useNetwork } from "wagmi";
+import { Chain, useAccount, useNetwork } from "wagmi";
 
 import { ToastContent } from "#/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/TransactionModal";
 import { Button } from "#/components";
 import Table from "#/components/Table";
 import { Toast } from "#/components/Toast";
 import { impersonateWhetherDAO, pools } from "#/lib/gql";
 import { tokenDictionary } from "#/utils/getTokenInfo";
-import { writeWithdrawInternalBalance } from "#/wagmi/withdrawInternalBalance";
+import {
+  usePrepareVaultManageUserBalance,
+  useVaultManageUserBalance,
+} from "#/wagmi/generated";
+
+export enum UserBalanceOpKind {
```

Comment:
would probably consolidate these constants and any helper functions at `lib` or `utils`